pages/linux/netperf.md

```diff
@@ -0,0 +1,24 @@
+# netperf
+
+> Client-side command for the network throughput benchmarking application netperf. Similar to iperf.
+> Homepage: <https://hewlettpackard.github.io/netperf/>.
+
+- Connect to server on a specific IP address via default port (12865):
+
+`netperf IP`
+
+- Specify port:
+
+`netperf IP -p 12345`
+
+- Specify number of seconds to run sampling (default is 10 seconds):
+
+`netperf IP -l SECONDS`
```

Comment:
Please use the [placeholder syntax](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#placeholder-syntax) for user-provided values. This allows the client to perform custom styling on these elements.

Reply:
Please check I use the placeholder syntax correctly, thanks.